### ISSUE TITLE
Don't overwrite offline products

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#1374]
Uploading an offline product, after the merge of don't overwrite product branch
![image](https://user-images.githubusercontent.com/1689815/38166403-7eeb821c-3523-11e8-8b5c-3cc052ff1d68.png)

**Comment by teolemon** [comment]
```March 31, 2018 at 8:34:45 PM CEST - teolemon Selected images (Changed: front_aa) - new image front_aa : 7.29 - view
March 31, 2018 at 8:34:45 PM CEST - teolemon Uploaded images (Added: 7) - new image 7 - image upload - view
March 31, 2018 at 8:34:35 PM CEST - teolemon Selected images (Added: front_aa, nutrition_aa, ingredients_aa) -- Data (Changed: lang, product_name, quantity, brands, product_name_fr) - (app)new android app - view```

**Comment by teolemon** [comment]
“https://world.*****/cgi/product_jqm2.pl?code=3700311800999&add_brands=XXXX&add_countries=en:france”

To try adding a brand without destructing the existing.

**Comment by huzaifaiftikhar** [comment]
Fixed in #1712
